src/components/Character.css

```diff
@@ -0,0 +1,20 @@
+.character-detail-name, .character-details {
+    font-family: 'Pathway Gothic One', sans-serif;
```

Comment:
We really liked attention to details and the beautiful UI you implemented 👍 

One small optimization here would be to set font-family on the body element and then you have it consistent in all places.

Reply:
App is now using 'Pathway Gothic One' on body.